### DISCUSSION TITLE
Are firmwares brand specific?

### DISCUSSION THREAD

**Discussion by ShadOLurk** [seaky/nspanel_pro_tools_apk#92] [General]
Just wondering if the sonoff firmwares on the googledrive link will work on other variants of the T6E panel? 
eg. I have a CK Panel Pro which I believe is hardware identical to the Sonoff.

**Comment by seaky** [comment]
It is an OTA update not a full fimrware, I would be suprised if you could install it.
The device has more partitions such as recovery, bootloader, user, and configurations etc.

The OTA update only contains the system/oem/vendor partitions. So even if you could install it there is a risk it might never start up again.
I have a full dump which was created by Rockchip AndroidTool but if you do not have any experience regarding how to flash a device I would not recommend it to try.

**Reply by ShadOLurk** [reply]
Thanks for the clarification,
tbh i don't think I need to update it (currently 2.0.0) , as I don't plan on using any of the native app features, I'm just running the HA companion app.
And hope to be able to use the built zigbee hub, with your next release of your "tools" app. :)
